public/build_config/webpack.prod.conf.js

```diff
@@ -18,19 +31,11 @@ module.exports = {
       }
     }),
     new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
-    new webpack.optimize.DedupePlugin(),
-    new webpack.optimize.OccurenceOrderPlugin(),
-    new webpack.optimize.UglifyJsPlugin({
```

Comment:
What are the implications of removing these plugins? Or are they perhaps the default behaviours of a production build now?

Reply:
As far as I understand it, this plugins no longer exist in the latest version of Webpack and (I think) their functionalities have been subsumed into the default behaviour of Webpack. I am not 100% on this - but if I'm wrong, I feel that we can get away with a sub-optimal webpack setup on an app like floodgate?

Reply:
In the case of Floodgate, that's probably a reasonable stance to take. I know it can be tricky to trace some of these kinds of changes in dependencies. Let's go with it and see how it fares.